### PR TITLE
antBuild: run default hooks

### PR DESCRIPTION
**Pull request by rasendubi**
###### Motivation for this change

Fixes https://github.com/NixOS/nixpkgs/issues/10931.
antBuild overrides buildPhase and installPhase without calling default
hooks.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

NOT TESTED

### CODE REVIEW

File: pkgs/build-support/release/ant-build.nix [hunk 1]
```diff
@@ -75,15 +79,19 @@ stdenv.mkDerivation (
...
       closeNest
     '' else stdenv.lib.concatMapStrings (t: ''
       header "Building '${t}' target"
-      ant ${antFlags} ${t} 
+      ant ${antFlags} ${t}
       closeNest
-    '') antTargets;
+    '') antTargets) ++ ''
+      runHook postBuild
+    '';;
```
Comment (copumpkin) [inline pkgs/build-support/release/ant-build.nix]: Extra semicolon!

Comment (rasendubi) [inline pkgs/build-support/release/ant-build.nix]: Thanks! fixed.

**Comment by garbas** [comment]
@rasendubi could you look why travis is failing?

**Comment by rasendubi** [comment]
Gotta go now. I believe someone is passing a string instead of list in `antTargets` but I don't have an evidence yet. Will look for it later.

**Comment by rasendubi** [comment]
Well, the build failed because of `junit`, which is marked as `broken`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/java/junit/default.nix#L26

**Comment by rasendubi** [comment]
Ok. It fails with different error before (test failure) and after the patch (nix evaluation failure). So I'm going to investigate to at least get this same error here.

**Comment by rasendubi** [comment]
Found it! I've confused `++` with `+` for string concatenation. Now junit should fail as expected ;)

**Comment by joachifm** [comment]
@rasendubi so that means this is ready to go, right?

**Comment by rasendubi** [comment]
@joachifm yep!

**Comment by joachifm** [comment]
Thank you

**Pull request merged**